Fix/missing icons

unlmtd opened the pull request:
Hey! I've kinda fixed the issue with the missing icons:

1. You couldn't update the icons package, because it is prohibited to write into the app bundle on native iOS. Due to a false error handling the app always said that the update was completed sucessfully, even if it didn't. I've changed the save location into `Documents`, so there's no problem anymore with the update.
For this change I also needed to change the `iNDSDBManager.h` a bit, but the DB gets copied at start of the app when it doesn't already exist and it gets opened automatically.

2. There's also a problem with the OpenVGDB. All icons get pulled from `http://img.gamefaqs.net`, but on further inspection I realized that GameFAQs moved the domain.
@FrederickGeek8 you need to alter the database and change all `http://img.gamefaqs.net/` to `http://gamefaqs1.cbsistatic.com/` and everything should work like a charm.

Image for reference:
![IMG_0228](https://user-images.githubusercontent.com/38921105/100543335-d4dd2500-324f-11eb-9e6f-66a066dc7e76.jpeg)

FrederickGeek8 commented:
Thanks for the pull request and nice catch with the OpenVGDB catch! I'll merge this request soon and hopefully, fix OpenVGDB after. If you're going for bonus points, creating a script (in a language of your choice) to change the URLs would be awesome. I'll run it on the server-side database.

Reminder (copied from the editor repo): You can find the latest database at https://inds.nerd.net/editor/openvgdb.sqlite

unlmtd commented:
Okay! I've added a script in Python to change all relevant entries inside the db.
I've tested it with a fresh downloaded database and changed the one inside of `Resources/`.

To test the changes without running the script server-side, you can just reset iNDS and after a restart, iNDS should provide the new db automatically.
